Clients/Xamarin.Interactive.Client/ClientApp.cs

```diff
@@ -161,27 +161,9 @@ protected ClientApp ()
         void PostAppSessionStarted ()
         {
             var appSession = new Telemetry.Models.AppSession {
```

Comment:
Do we even need a distinct app session event with everything now rolled into the context's global properties?

Reply:
Got it. I think we should allow posting arbitrary event names as strings that don't need to have any specific attached properties, so this sounds like a good first candidate.

I mostly wasn't sure if the act of initializing a new telemetry session would implicitly post an event like this. If we need one we need one.